doc/changes/devel/12326.other.rst

```diff
@@ -0,0 +1 @@
+Updated the text in the preprocessing tutorial to use :class:`mne.io.Raw.pick()` instead of the legacy :class:`mne.io.Raw.pick_types()`, by :newcontrib:`btkcodedev`.
```

Comment:
I'm a little surprised that these cross-refs worked; regardless, it's more accurate to mark these with the `:meth:` role since they're links to methods (not classes). The parentheses will get added automatically.

```suggestion
Updated the preprocessing tutorial to use :meth:`mne.io.Raw.pick` instead of the legacy :meth:`mne.io.Raw.pick_types`, by :newcontrib:`btkcodedev`.
```